apps/stripe/src/app/api/webhooks/stripe/stripe-webhook-response.ts

```diff
@@ -25,9 +29,7 @@ export class StripeWebhookErrorResponse {
   }
 
   getResponse() {
-    // todo What should be the message
-    return new Response("Error", {
-      // TODO What should be the error?
+    return new Response("Server error", {
```

Comment:
Same here, IMO `StripeWebhookRetryableErrorResponse` and `StripeWebhookNonRetryableErrorResponse` should be abstract and we should create actual errors with actual reasons extending them